apps/services/user-notification/src/app/modules/notifications/dto/notification.dto.ts

```diff
@@ -81,6 +84,10 @@ export class RenderedNotificationDto {
   @IsUUID()
   messageId!: string
 
+  @ApiProperty({ example: 'Þjóðskrá' })
+  @IsString()
+  senderId!: string | null | undefined
```

Comment:
`!`
`undefined`

It's definitely a string (or null or undefined) :laughing: 

---

Isn't this a bit strange?

Reply:
fixed